app/webpack/users/edit/components/relationships.jsx

```diff
@@ -0,0 +1,205 @@
+        </div>
+      </div>
+      <div className="col-md-2 col-sm-3 col-xs-4 margin-right-medium">
+        <div className="row flex-no-wrap search-margin">
+          <label className="margin-right" htmlFor="trusted">{I18n.t( "trusted" )}</label>
+          <select
+            className="form-control"
+            id="trusted"
+            name="trusted"
+            onChange={updateFilters}
+          >
+            <option value="all">{I18n.t( "all" )}</option>
+            <option value="yes">{I18n.t( "yes" )}</option>
+            <option value="no">{I18n.t( "no" )}</option>
+          </select>
+        </div>
+      </div>
+      <div className="col-md-3 col-sm-4 col-xs-4">
+        <div className="row flex-no-wrap">
+          <label className="margin-right" htmlFor="sort_by">{I18n.t( "sort_by" )}</label>
+          <select
+            className="form-control"
+            id="sort_by"
+            name="sort_by"
+            onChange={sortRelationships}
+          >
+            <option value="recently_added">{I18n.t( "recently_added" )}</option>
+            <option value="earliest_added">{I18n.t( "earliest_added" )}</option>
+            <option value="a_to_z">{I18n.t( "a_to_z" )}</option>
+            <option value="z_to_a">{I18n.t( "z_to_a" )}</option>
+          </select>
+        </div>
+      </div>
+    </div>
+  );
+
+  const showEmptyState = ( ) => (
+    <SettingsItem>
+      <p className="bold">
+        {I18n.t( "youre_not_following_anyone_on_inat", { site_name: SITE.name } )}
+      </p>
+    </SettingsItem>
+  );
+
+  return (
+    <div>
+      <SettingsItem>
+        <h4>{I18n.t( "relationships_user_settings" )}</h4>
+        {showFilters( )}
+      </SettingsItem>
+      {relationships.length === 0 && showEmptyState( )}
+      <div className={relationships.length === 0 ? "hidden" : null}>
+        <div className="row hidden-xs">
+          <div className="col-xs-4">
+            <label>{I18n.t( "name" )}</label>
+          </div>
+          <div className="col-sm-8">
+            <label>{I18n.t( "actions" )}</label>
+          </div>
+        </div>
+        {filteredRelationships.length > 0 && showRelationships( )}
+        <div className="divider" />
+        <div className="Pagination text-center">
+          <Pagination
+            total={filteredRelationships.length}
+            current={page}
+            pageSize={10}
+            onChange={p => loadPage( p )}
+          />
+        </div>
+      </div>
+      <div className="row">
+        <BlockedMutedUsersContainer
+          headerText={I18n.t( "blocked_users" )}
+          id="blocked_users"
+          placeholder={I18n.t( "add_blocked_users" )}
+          buttonText={I18n.t( "unblock" )}
+          htmlDescription={{
+            __html: I18n.t( "views.users.edit.blocking_desc_html", {
+              site_name: SITE.name,
+              // noting that this help_email isn't populating
+              help_email: "help@inaturalist.org" // SITE.help_email
```

Comment:
Use `SITE.help_email` anyway. You can configure it locally at http://localhost:3000/sites/1/edit, but in production, different sites may have different help email addresses. If we want to ensure people are using *our* help address, we can do something later like `DEFAULT_SITE.help_email`.

Reply:
Fixed.